xml/System.Security.Cryptography.X509Certificates/X509ChainElementCollection.xml

```diff
@@ -414,8 +414,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a valie indicating whether access to the collection is synchronized (thread safe).</summary>
+        <value><see langword="true"/> if access to the collection is synchronized (thread safe); otherwise <see langword="false"/>. The default is <see langword="false"/>.</value>
```

Comment:
```suggestion
        <value><see langword="true"/> if access to the collection is synchronized (thread safe); otherwise, <see langword="false"/>. The default is <see langword="false"/>.</value>
```